PyInstaller/depend/dylib.py

```diff
@@ -34,7 +37,15 @@
 # TODO convert _excludes and includes into type 'set()'.
```

Comment:
Could you do implement this TODO and remove the '1' as values in the patterns?  the dict() type was used for compatibility with python 2.4 and older. But we no longer support these old versions and '1' has no other meaning anymore.